### ISSUE TITLE
Missing `dc` namespace.

### ISSUE THREAD

**Issue by ielis** [bio-ontology-research-group/unit-ontology#52]
Dear UO developers, I would like to report that a declaration of `dc` namespace is missing in the `uo.owl`.

I am trying to use UO in a project, and when running the following commands:
```shell
curl --output uo.owl --location http://purl.obolibrary.org/obo/uo.owl
robot convert --input uo.owl  --format owl --output uo.local.owl
```
I receive the error:
```
[Fatal Error] :7563:75: The prefix "dc" for element "dc:creator" is not bound.
INVALID ONTOLOGY FILE ERROR Could not load a valid ontology from file: uo.owl
```

Apparently, the `dc` namespace declaration is missing from `uo.owl`, because adding the following line fixes the error:
```xml
xmlns:dc="http://purl.org/dc/elements/1.1/"
```

Do you think this should be fixed on UO side or it is a parsing issue? Thanks a lot!

> Note: `robot` refers to [ROBOT](https://robot.obolibrary.org/) `v1.8.3`, and the no-op `robot convert` from OWL to OWL format above just demonstrates the error.

**Comment by udp** [comment]
I just hit the same issue. It's definitely not a parsing issue. Without the namespace in the header it's invalid RDF/XML.

**Comment by udp** [comment]
Does not parse with Jena for instance:

    org.apache.jena.riot.RiotException: [line: 7563, col: 75] The prefix "dc" for element "dc:creator" is not bound.
	    at org.apache.jena.riot.system.ErrorHandlerFactory$ErrorHandlerStd.fatal(ErrorHandlerFactory.java:153)
	    at org.apache.jena.riot.lang.ReaderRIOTRDFXML$ErrorHandlerBridge.fatalError(ReaderRIOTRDFXML.java:313)
	    at org.apache.jena.rdfxml.xmlinput.impl.ARPSaxErrorHandler.fatalError(ARPSaxErrorHandler.java:47)
	    at org.apache.jena.rdfxml.xmlinput.impl.XMLHandler.warning(XMLHandler.java:199)
	    at org.apache.jena.rdfxml.xmlinput.impl.XMLHandler.fatalError(XMLHandler.java:229)

**Comment by bkr-iotic** [comment]
Hello, I have raised a PR to fix this #53

**Comment by bkr-iotic** [comment]
The same commit introduced other parsing errors which are fixed in #55

**Comment by reality** [comment]
Hi, my read of this is that this is fixed by the pull request. Please feel free to re-open if I'm incorrect. Thanks a lot!
